app/views/articles/_video_player.html.erb

```diff
@@ -1,16 +1,30 @@
-<div itemscope itemtype="http://schema.org/VideoObject" class="video-player-header">
+<div class="video-player-header">
   <% if meta_tags %>
-    <meta itemprop="uploadDate" content="<%= article.published_at %>" />
-    <meta itemprop="name" content="<%= article.title %>" />
-    <meta itemprop="description" content="<%= article.description %>" />
-    <meta itemprop="thumbnailUrl" content="<%= cloudinary(article.video_thumbnail_url, 880) %>" />
-    <meta itemprop="contentUrl" content="<%= article.video_source_url %>" />
+    <meta content="<%= article.published_at %>" />
+    <meta content="<%= article.title %>" />
+    <meta content="<%= article.description %>" />
+    <meta content="<%= cloudinary(article.video_thumbnail_url, 880) %>" />
+    <meta content="<%= article.video_source_url %>" />
     <% minutes, seconds = article.video_duration_in_minutes.split(":") %>
-    <meta itemprop="duration" content="<%= format("PT%<minutes>sM%<seconds>sS", minutes: minutes, seconds: seconds) %>" />
+    <meta content="<%= format("PT%<minutes>sM%<seconds>sS", minutes: minutes, seconds: seconds) %>" />
```

Comment:
I have the feeling that these meta tags, now that they don't have a `itemprop` anymore, can actually be removed. They lack purpose I guess.

cc @benhalpern